### PR TITLE
Bashless scenario1, 2, 3, 4

### LINKED ISSUE #205: Bashless scenario 1

### LINKED ISSUE #206: Bashless scenario 3

### LINKED ISSUE #214: Bashless scenario 2

### LINKED ISSUE #215: Bashless scenario 4

### PR DESCRIPTION
**Pull request by Polina-Gubina**
Bashless scenario1, 2, 3, 4
Add destroy for scenario1_5
Change role for installation terraform 

Resolve #205
Resolve #214
Resolve #206  
Resolve #215 

Resolve opentelekomcloud-infra/csm-ansible-roles#25

### CODE REVIEW

File: scenarios/scenario2/resources/vm.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 resource "opentelekomcloud_compute_keypair_v2" "pair" {
-  name       = "${var.prefix}-kp"
+  name       = "${var.prefix}-kl"
```
Comment (anton-sidelnikov) [inline scenarios/scenario2/resources/vm.tf:2]: kp

File: playbooks/scenario1_setup.yml [hunk 2]
```diff
@@ -69,3 +139,29 @@
...
+      ignore_errors: yes
+
+- name: Start reporting for scenario
+  hosts: test_host
+  tasks:
+  - name: Execute csm_test_utils  # noqa 301 305
+    shell: "nohup python3 -m csm_test_utils monitor --target {{ loadbalancer_public_ip }} --telegraf 'http://{{ bastion_public_ip }}' &"
+
+
+
```
Comment (anton-sidelnikov) [inline playbooks/scenario1_setup.yml:166]: kill it

File: playbooks/scenario2_setup.yml [hunk 3]
```diff
@@ -71,3 +106,26 @@
                   proxy_pass         http://localhost:{{ telegraf_local_port }};
                   proxy_set_header   Host $host;
               }
+
+- name: Checking delatore processes
```
Comment (anton-sidelnikov) [inline playbooks/scenario2_setup.yml:110]: rename it

File: playbooks/scenario2_setup.yml [hunk 4]
```diff
@@ -71,3 +106,26 @@
...
+  hosts: test_host
+  tasks:
+    - name: Get running delatore processes list from remote host  # noqa 301 305
```
Comment (anton-sidelnikov) [inline playbooks/scenario2_setup.yml:113]: not delatore

File: playbooks/scenario2_setup.yml [hunk 5]
```diff
@@ -71,3 +106,26 @@
...
+      ignore_errors: yes
+      shell: "ps aux | grep 'csm_test_utils rds_monitor' | awk '{print $2}'"
+      register: process_monitor
+    - name: Kill running delatore processes  # noqa 301 305
```
Comment (anton-sidelnikov) [inline playbooks/scenario2_setup.yml:117]: not delatore

File: playbooks/scenario2_setup.yml [hunk 6]
```diff
@@ -71,3 +106,26 @@
...
+      wait_for:
+        path: "/proc/{{ process_monitor.stdout_lines[0] }}/status"
+        state: absent
+      ignore_errors: yes
+
+- name: Run reporting for scenario
+  hosts: test_host
+  tasks:
+  - name: Execute csm_test_utils  # noqa 301 305
+    shell: "nohup python3.7 -m csm_test_utils rds_monitor --target {{ scn2_public_ip }} --telegraf 'http://{{ scn2_public_ip }}' &"
```
Comment (anton-sidelnikov) [inline playbooks/scenario2_setup.yml:130]: python3

File: playbooks/scenario1_setup.yml [hunk 7]
```diff
@@ -69,3 +139,29 @@
             - "enp0s[0-1]"
             - "lo"
             - "ens3"
+
+- name: Checking reporting processes
+  hosts: test_host
+  tasks:
+    - name: Get reporting delatore processes list from remote host  # noqa 301 305
```
Comment (anton-sidelnikov) [inline playbooks/scenario1_setup.yml:146]: not delatore

File: playbooks/scenario3_setup.yml [hunk 8]
```diff
@@ -1,10 +1,61 @@
 ---
+- name: Prepare clouds.yaml for test_host
+  import_playbook: prepare_config.yml
+
+- name: Prepare variables
+  hosts: test_host
+  tasks:
+  - name: Register scenario 1 vars
```
Comment (lego963) [inline playbooks/scenario3_setup.yml:8]: missprint

File: playbooks/scenario1_setup.yml [hunk 9]
```diff
@@ -1,4 +1,74 @@
...
+      ansible_host: "{{ bastion_public_ip }}"
+      ansible_ssh_user: "linux"
+      ansible_ssh_private_key_file: "{{ local_private_key }}"
+  - name: Register nodes
+    add_host:
+      name: "{{ item }}"
+      groups: "gatewayed"
+      ansible_ssh_user: "linux"
+      ansible_ssh_private_key_file: "{{ local_private_key }}"
+      ansible_ssh_common_args: "-o ProxyCommand='ssh -W %h:%p -q linux@{{ bastion_public_ip }} -i {{ local_private_key }}' -o PasswordAuthentication=no"
```
Comment (outcatcher) [inline playbooks/scenario1_setup.yml:53]: Let's try to move it to group vars

File: playbooks/scenario2_setup.yml [hunk 10]
```diff
@@ -1,12 +1,47 @@
...
+      terraform_base_dir: "terraform"
+      requirements: "requirements.txt"
+
+- name: Build infrastructure
+  hosts: test_host
+  roles:
+    - build_infrastructure
+  environment:
+    OS_CLIENT_CONFIG_FILE: "{{ clouds_yaml_file }}"
+    TF_VAR_psql_password: "{{ psql_password }}"
```
Comment (outcatcher) [inline playbooks/scenario2_setup.yml:19]: Can it be done using Ansible vars, not env vars?

File: playbooks/scenario2_destroy.yml [hunk 11]
```diff
@@ -0,0 +1,15 @@
+---
+- name: Prepare clouds.yaml for test_host
+  import_playbook: prepare_config.yml
+
+- name: Prepare variables
+  hosts: test_host
+  environment:
+    OS_CLIENT_CONFIG_FILE: "{{ clouds_yaml_file }}"
+    TF_VAR_psql_password: "{{ psql_password }}"
```
Comment (outcatcher) [inline playbooks/scenario2_destroy.yml:9]: Same as in `_setup` playbook

File: scenarios/scenario4/resources/variables.tf [hunk 12]
```diff
@@ -8,7 +8,7 @@ variable "router_id" {}
 variable "bastion_local_ip" {}
 variable "bastion_eip" {}
 variable "nodes_count" {}
-variable "prefix" {}
+variable "prefix" { default = "prefix" }
```
Comment (outcatcher) [inline scenarios/scenario4/resources/variables.tf:11]: Why default prefix is not empty, for example?

File: scenarios/scenario4/resources/smn.tf [hunk 13]
```diff
@@ -33,7 +33,7 @@ resource "opentelekomcloud_ces_alarmrule" "instances_count" {
 }
 
 resource "opentelekomcloud_smn_topic_v2" "instances_count_topic" {
-  name            = "${var.prefix}_instances_count_topic"
+  name            = "scn4_instances_count_topic"
```
Comment (anton-sidelnikov) [inline scenarios/scenario4/resources/smn.tf:36]: need to set ${var.prefix}_instances_count_topic as in autoscaling.tf

File: scenarios/scenario4/resources/smn.tf [hunk 14]
```diff
@@ -1,5 +1,5 @@
 resource "opentelekomcloud_ces_alarmrule" "instances_count" {
-  alarm_name = "${var.prefix}_instances_count"
+  alarm_name = "scn4_instances_count"
```
Comment (anton-sidelnikov) [inline scenarios/scenario4/resources/smn.tf:2]: need to set ${var.prefix}_instances_count_topic as in autoscaling.tf

File: scenarios/scenario4/resources/vm.tf [hunk 15]
```diff
@@ -3,10 +3,24 @@ data "opentelekomcloud_images_image_v2" "current_image" {
...
+    subnet_id  = var.subnet_id
+    ip_address = "${var.net_address}.${count.index + 10}"
+  }
+}
+
 # Create instance
 resource "opentelekomcloud_compute_instance_v2" "http" {
   count             = var.nodes_count
-  name              = "${local.workspace_prefix}${var.prefix}_basic_${count.index}"
+  name              = "${var.prefix}_basic_${count.index}"
```
Comment (outcatcher) [inline scenarios/scenario4/resources/vm.tf:23]: Why remove workspace prefix?

File: scenarios/scenario4/resources/smn.tf [hunk 16]
```diff
@@ -42,3 +42,9 @@ resource "opentelekomcloud_smn_subscription_v2" "subscription" {
   endpoint        = "http://${var.bastion_eip}/smn"
   protocol        = "http"
 }
+
+resource "opentelekomcloud_smn_subscription_v2" "subscription_v2" {
```
Comment (outcatcher) [inline scenarios/scenario4/resources/smn.tf:46]: Duplicating with previous resource?

**Pull request merged**